lms/static/sass/_pearsonx.scss

```diff
@@ -304,11 +304,25 @@ section.course-info {
       }
     }
   }
+
+  .details {
+    @media screen and (max-width: 760px) {
```

Comment:
For each new rule introduced here, please add a comment (starting with `//` to exclude it from the CSS) that includes a link to the upstream PR and says we can remove this rule when that PR is merged.

Reply:
Done, and squashed with previous commit.